src/PagerAndroid.js

```diff
@@ -122,12 +119,15 @@ export default class PagerAndroid<T: *> extends React.Component<Props<T>> {
   };
 
   render() {
-    const {
-      children,
-      navigationState,
-      swipeEnabled,
-      keyboardDismissMode,
-    } = this.props;
+    const { navigationState, swipeEnabled, keyboardDismissMode } = this.props;
+    let children = [];
+    React.Children.forEach(this.props.children, c => {
```

Comment:
Isn't this `.forEach` function the same as `.reverse`, which we were doing previously with `content.reverse()`?

Reply:
Yeah, but doesn't that mean we just need to reverse before mapping and not after? Because what you're doing with `forEach` is just `reverse`:

```js
var items = ['a', 'b', 'c'];
var result = [];

items.forEach(c => result = [c, ...result]);

console.log(result); // ["c", "b", "a"]

items.reverse();

console.log(items); // ["c", "b", "a"]
```